### DISCUSSION TITLE
part-ii/2018-15/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#39] [Question comments]
Part II 2018 Applied Probability: Paper 4, Section II, J 

https://questions.tripos.org/part-ii/2018-15/

**Comment by ArthurConmy** [comment]
![](https://i.imgur.com/G4rb2Nh.png)

![](https://i.imgur.com/b4sUO4W.png)

For the final part, let $e_i$ be the expected number of further keys to press given that we've pressed the first $i$ keys of "lava", so $e_4=0$ and we seek $e_0$. Then we have the simultaneous equations $e_0 = 1+ \frac{1}{100} e_1 + \frac{99}{100} e_0$ and for $ 1 \le i \le 3, e_i = 1 + \frac{1}{100}e_{i+1} + \frac{1}{100} e_1 + \frac{98}{100} e_0$. These solve to give $\boxed{ 10^8 }$ for "lava", and the same method gives $\boxed{10100}$ for "aa".

**Reply by DannyDuberstein** [reply]
Don't think this is correct

**Reply by ArthurConmy** [reply]
Thanks: the correct equations are $ e_i = 1 + \frac{1}{100} e_{i+1} + \frac{99}{100} e_0 $ but weirdly I think the numerical values are correct ... ?

**Reply by DannyDuberstein** [reply]
I got 100^4 for lava

**Reply by DannyDuberstein** [reply]
I think those eqns aren't right either as conditional on having just broken the chain you are more likely to have an "l" so it's not e_0

**Reply by ArthurConmy** [reply]
Dear me, this doesn't bode well for exams but yes I do now agree, thanks @DannyDuberstein
